client/components/progress-bar/README.md

```diff
@@ -26,4 +26,5 @@ render: function() {
 * `total`: a number representing the value corresponding to the 100% of the bar (optional, default == 100).
 * `color`: a string of a css color (optional).
 * `title`: a string for the title attribute (optional).
+* `isCompact`: If true, displays as a compact progress bar (optional).
```

Comment:
More recently, we've shied away from prefixing boolean types with `is`. For example, see [`<Button compact />`](https://github.com/Automattic/wp-calypso/blob/master/client/components/button/index.jsx). My argument for this is to better align with DOM attributes which are similarly not prefixed (e.g. `disabled`, `readOnly`).